engine/polyglot-api/src/main/scala/org/enso/polyglot/runtime/Runtime.scala

```diff
@@ -434,9 +446,12 @@ object Runtime {
         /** Indicates that the expression was computed to a value.
           *
           * @param warnings information about attached warnings.
+          * @param functionSchema the function schema
```

Comment:
```suggestion
          * @param functionSchema if the value represents a function, the function schema of that function, empty option otherwise 
```